.github/workflows/testing-release.yml

```diff
@@ -0,0 +1,97 @@
+name: Testing Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Monika NPM version to create docker image'
+        required: true
+        type: string
+
+jobs:
+  testing:
+    name: Testing Release
+    if: github.repository == 'hyperjump/monika-nightly'
+    runs-on: ubuntu-latest
+    outputs:
+      NEXT_VERSION: ${{ steps.version.outputs.NEXT_VERSION }}
+    steps:
+      - name: 🛑 Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
+
+      - name: ⬇️ Checkout repo
+        uses: actions/checkout@v3
+
+      - name: 🔧 Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - name: 📥 Install Dependencies
+        run: npm ci
+
+      - name: 🧪 Run Unit Tests
+        run: npm test
+
+      - name: ⤴️ Switch Version based on inputs
+        id: version
+        run: |
+          git config --local user.email "monika@hyperjump.tech"
+          git config --local user.name "Monika Bot"
+          git fetch --all --tags
+          SHA=$(git rev-parse v${{ inputs.version }})
```

Comment:
This will fail if I put "v0.0.0-nightly-20220927-44f4ce5". You don't need to append "v" at the beginning.